doc/yaml_config.md

```diff
@@ -28,6 +28,40 @@ Software manager related options. It is map with following keys:
 
 Array of url for installation repositories.
 
+#### mandatory\_patterns
+
+Array of patterns that have to be selected.
+
+#### optional\_patterns
+
+Array of patterns that should be selected but can be deselected or skipped if not available.
+
+### security
+
+Options related to security
+
+#### lsm
+
+Default linux security module. Currently supported values are `selinux`, `apparmor` and `none`.
+
+#### available\_lsms
+
+Map for available linux security modules. If only one module is
+available it means that lsm is not configurable.
+
+##### patterns
+
+Required patterns for given lsm to be selected.
+
+##### policy
```

Comment:
Is this not the same than "selinux_policy" below?

Reply:
yeah, my fault. I consider both options and looks like I forget it there. Thanks for notice it.